Launcher/Classes/PlistOperations.swift

```diff
@@ -3,10 +3,8 @@ import Foundation
 class PlistOperations {
     let plistPath: String
     let fileManager = FileManager.default
-    let dictionaryKey: String
     
-    public init(forKey key: String, defaultPlistPath path: String = "/CalabashLauncherSettings.plist") {
-        dictionaryKey = key
+    public init(defaultPlistPath path: String = "/CalabashLauncherSettings.plist") {
```

Comment:
As `plistPath` is a `let`, I would rename this to simply `path`; not `defaultPlistPath`.

Reply:
Maybe put the string in `Constants.swift`?